src/modules/dashboard/components/Extensions/ExtensionDetails.tsx

```diff
@@ -184,6 +207,9 @@ const ExtensionDetails = ({
     !(extesionCanBeInstalled || extesionCanBeEnabled) &&
     latestNetworkExtensionVersion > extension.currentVersion;
 
+  // TEMP flag
+  const shouldShowWarning = false;
```

Comment:
In your PR description you say that the temp flag should be true. Which version is correct?

Reply:
Yes, I will wire this via https://linear.app/colony/issue/DEV-365/wire-up-incompatible-extension-warning

This issue is UI only